app/frontend/src/features/auth/signup/CompleteSignupForm.tsx

```diff
@@ -129,11 +127,12 @@ export default function CompleteSignupForm() {
   }, [urlToken, authActions, location.pathname, setValue, history]);
 
   const completeSignup = handleSubmit(async (data: SignupInputs) => {
-    if (Object.entries(data.location).length === 0) {
-      setIsLocationEmpty(true);
+    if ((data.location?.address ?? "") === "") {
+      authActions.authError(SIGN_UP_LOCATION_MISSING);
       return;
     }
 
+    // authActions catches errors here
```

Comment:
oh lol

Reply:
But is `authState.error` displayed anywhere?

Reply:
In the parent!